CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ UNRELEASED
 * [ [#1402](https://github.com/digitalfabrik/integreat-cms/issues/1402) ] Also duplicate imprints for new regions
 * [ [#1408](https://github.com/digitalfabrik/integreat-cms/issues/1408) ] Remove duplication of push API tokens for pages during duplication process
 * [ [#1404](https://github.com/digitalfabrik/integreat-cms/issues/1404) ] Fix performance issue for select all on huge page trees
+* [ [#1413](https://github.com/digitalfabrik/integreat-cms/issues/1413) ] Fix change of pagination size in broken link checker
+* [ [#1422](https://github.com/digitalfabrik/integreat-cms/issues/1422) ] Keep page in broken link checker when performing replacement
```

Comment:
I'm not sure if I understand this line correctly. The broken URL is removed from the list of broken links as soon as it is replaced.

Reply:
Yeah, it's ambiguous, in this case I meant the pagination page... I'll adapt the wording.